frontend/src/component/admin/billing/BillingDashboard/BillingPlan/BillingPlan.tsx

```diff
@@ -9,16 +9,14 @@ import { Badge } from 'component/common/Badge/Badge';
 import { BillingDetails } from './BillingDetails';
 import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
 
-export const BILLING_PLAN_PRICES: Record<string, number> = {
-    [InstancePlan.PRO]: 80,
-};
+export const BILLING_PRO_BASE_PRICE = 80;
+export const BILLING_PRO_SEAT_PRICE = 15;
+export const BILLING_PAYG_SEAT_PRICE = 75;
+export const BILLING_TRAFFIC_PRICE = 5;
```

Comment:
I'd love to get rid of these static constants but I kept them for now as defaults, in case we have any mistakes in our logic.

We can remove them once we feel more confident.